proposals/lip-0014.md

```diff
@@ -471,6 +477,9 @@ Assume the mechanism described in this section is triggered according to the con
 (A.height<B.height && A.heightPrevoted==B.heightPrevoted)
 ```
 
+It is again essential that a node does not forge any blocks while performing the following steps for switching to a different chain. Otherwise, the node may forge a block contradicting the fork choice rule. This means that forging must be deactivated before starting the following steps.
+After exiting the "Block Synchronization Mechanism" (because it is aborted or successfully completed) the block at the tip of the chain satisfies the condition above and forging can be activated again.
```

Comment:
It mentions `Block Sync Mechanism` but this is also valid for `Fast Chain Switching`, right? Whenever the FCS finishes successfully, forging is activated again. If it fails, I assume `block synchronization` in.

Reply:
An analogous sentence is contained in the `Fast Chain Switching Mechanis` section. I don't understand the last sentence.